website/docs/pipelines/promoting-applications.mdx

```diff
@@ -295,7 +376,7 @@ allow access to the single repo that your configuration manifests exist.
 ![create token least priviledge](img/fine-grained-token.png)
  </details>
 
-5. **Review active access tokens on a regular basis**: to ensure that only the ones that are required are present at all times.
+4. **Review active access tokens on a regular basis**: to ensure that only the ones that are required are present at all times.
```

Comment:
extend with following guidance for git provider and need to update the secret after it expires.

Reply:
updated